src/NServiceBus.Core/Encryption/Encryptor.cs

```diff
@@ -51,7 +51,7 @@ bool VerifyPrerequisite(FeatureConfigurationContext context)
         static List<PropertyInfo> GetEncryptedProperties(FeatureConfigurationContext context)
         {
             var conventions = context.Settings.Get<Conventions>();
-            return context.Settings.GetAvailableTypes()
+            return context.Settings.GetAvailableMessageTypes()
```

Comment:
This appears to be the one place in this PR that wasn't previously using `IsMessageType`, so I'm assuming it is actually desirable to scope this to message types here as well?